client/src/Display/Display.js

```diff
@@ -8,11 +8,15 @@ class Display extends Component {
     return (
       <>
         <div className="display">
-          <h2>Player At Bat</h2>
-          <p>Balls: {this.props.state.balls}</p>
-          <p>Strikes: {this.props.state.strikes}</p>
-          <p>Fouls: {this.props.state.fouls}</p>
-          <p>Hits: {this.props.state.hits}</p>
+          <h2 data-testid="title">Player At Bat</h2>
+          <label>Balls:</label>
+          <p data-testid="balls">{this.props.balls}</p>
+          <label>Strikes:</label>
+          <p data-testid="strikes">{this.props.strikes}</p>
+          <label>Fouls:</label>
+          <p data-testid="fouls">{this.props.fouls}</p>
+          <label>Hits:</label>
+          <p data-testid="hits">{this.props.hits}</p>
```

Comment:
These labels and p tags are pretty repetitive; you could make them iteratively in this way:
```javascript
<h2 data-testid="title">Player At Bat</h2>
{[ 'balls', 'strikes', 'fouls', 'hits' ].map(name => (
  <>
    <label>{name[0].toUpperCase() + name.slice(1)}:</label>
    <p data-testid={name}>{this.props[name]}</p>
  </>
))}
```